### PR TITLE
Feature/25589 staff generate pins try it out load test

### PR DESCRIPTION
**Pull request by swapnaravula**
Added below load test scenarios
i) teacher-generate-pins-try-it-out.jmx - feature/25589
ii teacher-edit-pupil.jmx - feature/25583

### CODE REVIEW

File: load-test/scenarios/teacher-edit-pupil/teacher-edit-pupil.jmx [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+hmap.put(15, &quot;R&quot;);
+hmap.put(16, &quot;T&quot;);
+hmap.put(17, &quot;U&quot;);
+hmap.put(18, &quot;V&quot;);
+hmap.put(19, &quot;W&quot;);
+hmap.put(20, &quot;X&quot;);
+hmap.put(21, &quot;Y&quot;);
+hmap.put(22, &quot;Z&quot;);
+
+String upnNuunber= hmap.get(checkLetterIndex) + upnDigits;
```
Comment (GuyHarwood) [inline load-test/scenarios/teacher-edit-pupil/teacher-edit-pupil.jmx]: should this be `upnNumber` ?

File: load-test/scenarios/teacher-edit-pupil/teacher-edit-pupil.jmx [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+              <sentBytes>true</sentBytes>
+              <threadCounts>true</threadCounts>
+              <idleTime>true</idleTime>
+              <connectTime>true</connectTime>
+            </value>
+          </objProp>
+          <stringProp name="filename">resultsTree.csv</stringProp>
+        </ResultCollector>
+        <hashTree/>
+        <JSR223Listener guiclass="TestBeanGUI" testclass="JSR223Listener" testname="JSR223 Listener" enabled="false">
```
Comment (GuyHarwood) [inline load-test/scenarios/teacher-edit-pupil/teacher-edit-pupil.jmx]: what is this for? its name doesnt tell me anything about it

File: load-test/scenarios/teacher-edit-pupil/teacher-edit-pupil.jmx (hunk 1 above)
Comment (GuyHarwood) [reply load-test/scenarios/teacher-edit-pupil/teacher-edit-pupil.jmx]: also, are we sure this script always generates a valid UPN?

Comment (swapnaravula) [reply load-test/scenarios/teacher-edit-pupil/teacher-edit-pupil.jmx]: Yes, this is to generate UPN and yes I am sure it work always as I tested it several times with many users.

File: load-test/scenarios/teacher-edit-pupil/teacher-edit-pupil.jmx (hunk 2 above)
Comment (swapnaravula) [reply load-test/scenarios/teacher-edit-pupil/teacher-edit-pupil.jmx]: This is some thing to log the error codes and their response messages while debugging. I have disabled it to make it an optional.

**Pull request merged**